### PR TITLE
Deploy droplets

### PR DESCRIPTION
**Pull request by sakisv**
Adds two scripts:

1. `scripts/deploy.sh` which uses the CF API to reuse droplets and trigger deployments
2. `scripts/create-droplet.sh` which is a helper script to run locally if you need to deploy something to preview

Best reviewed commit-by-commit.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,9 +135,7 @@ cf-deploy: ## Deploys the app to Cloud Foundry
 	# generate manifest (including secrets) and write it to CF_MANIFEST_PATH (in /tmp/)
 	make -s CF_APP=${CF_APP} generate-manifest > ${CF_MANIFEST_PATH}
 
-	# fails after 15 mins if deploy doesn't work
-	# reads manifest from CF_MANIFEST_PATH
-	CF_STARTUP_TIMEOUT=15 cf push ${CF_APP} --strategy=rolling -f ${CF_MANIFEST_PATH}
+	CF_APP=${CF_APP} CF_MANIFEST_PATH=${CF_MANIFEST_PATH} ./scripts/deploy.sh
```
Comment (rjbaker) [inline Makefile:138]: Presumably this change will begin the switch to the new deployment method in all environments?

**Review by rjbaker — APPROVED** [review]
Looks good and I understand the droplet build/copy/deploy process following our conversations. One question above about rolling this out.

File: Makefile (hunk 1 above)
Comment (sakisv) [reply Makefile:138]: Correcty. Do you think it warrants a preview+staging rollout first?

Comment (sakisv) [reply Makefile]: Addressed in 8176014

**Pull request merged**